mbus-api/src/lib.rs

```diff
@@ -214,6 +220,55 @@ pub type BusOptions = nats::Options;
 /// Save on typing
 pub type DynBus = Box<dyn Bus>;
 
+/// Timeout for receiving a reply to a request message
+/// Max number of retries until it gives up
+#[derive(Clone)]
+pub struct TimeoutOptions {
+    /// initial request message timeout
+    pub(crate) timeout: std::time::Duration,
+    /// max number of retries following the initial attempt's timeout
+    pub(crate) max_retries: Option<u32>,
+}
+
+impl TimeoutOptions {
+    pub(crate) fn default_timeout() -> Duration {
+        Duration::from_secs(6)
```

Comment:
😋 I think indeed that after careful experimentation has demonstrated it's best to use 6.